tests/functional/persist_docs_tests/test_persist_docs.py

```diff
@@ -40,8 +70,12 @@ def _assert_has_table_comments(self, table_node):
             table_comment, table_id_comment, table_name_comment
         )
 
-    def _assert_has_view_comments(self, view_node, has_node_comments=True,
-                                  has_column_comments=True):
+    def _assert_has_view_comments(
```

Comment:
This and `_assert_has_table_comments()` look like a good use case for parameterized tests (`@pytest.mark.paraterize(...)`. I think that's what's effectively happening here, but it could be maintained more easily with the pytest built-ins.

Reply:
I looked into this, and agree it's a neat built in. But I'm not sure it matches this use case. This flag appears to accomodate fixed values whereas we're passing in dynamically generated nodes. Am I missing something from your thinking here?